plugins/module_utils/system_access_users_utils.py

```diff
@@ -534,6 +603,23 @@ def from_ansible_module_params(cls, params: dict) -> "User":
 
         return cls(**user_dict)
 
+    @staticmethod
+    def _apikeys_from_xml(apikeys: dict) -> List[Dict]:
+
+        if isinstance(apikeys, str):
+            return [{}]
+
+        api_keys = []
+        if isinstance(apikeys, list):
+            for item in apikeys:
+                item = item.get("item", {})
+                api_keys.append({"key": item.get("key"), "secret": item.get("secret")})
```

Comment:
Maybe renaming the loop variable might protect against some future side effects regarding object references:

```suggestion
            for pair in apikeys:
                item = pair.get("item", {})
                api_keys.append({"key": item.get("key"), "secret": item.get("secret")})
```

Reply:
thanks for the suggestion! Fixed in a3e6152